package.json

```diff
@@ -7,7 +7,9 @@
     "node": ">= 6.10.0"
   },
   "config": {
-    "jsSrc": "src/"
+    "jsSrc": "src/",
+    "chromeZipFileName": "chrome-headless-lambda-linux-60.0.3095.0.zip",
```

Comment:
You might need to move `chromeZipFileName` and `nssZipFileName` into the `package.json` within the `packages/lambda` folder. Only the contents within the `packages/lambda` folder get published in the NPM package.